src/OmniSharp/Api/ProjectSystem/OmnisharpController.CurrentProject.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.AspNet.Mvc;
+using OmniSharp.AspNet5;
+using OmniSharp.Models;
+using OmniSharp.MSBuild;
+using System.Linq;
+
+namespace OmniSharp
+{
+    public class CurrentProjectController
+    {
+        private readonly AspNet5Context _aspnet5Context;
+        private readonly OmnisharpWorkspace _workspace;
+        private readonly MSBuildContext _msbuildContext;
+
+        public CurrentProjectController(AspNet5Context aspnet5Context, MSBuildContext msbuildContext, OmnisharpWorkspace workspace)
+        {
+            _aspnet5Context = aspnet5Context;
+            _msbuildContext = msbuildContext;
+            _workspace = workspace;
+        }
+
+        [HttpPost("currentprojectcommands")]
```

Comment:
Yea, I don't think this controller makes any sense. It makes more sense to do what I said above. Use the controller that returns project information to allow returning individual project information. You can also have something that returns the current project.